ote/src/clj/ote/services/transport.clj

```diff
@@ -103,7 +92,9 @@
       (http/no-cache-transit-response
         (-> (assoc ts ::t-service/operation-area
                       (places/fetch-transport-service-operation-area db id))
-            (append-ote-netex-interfaces config db))
+            vector
+            (netex-util/append-ote-netex-urls config db ::t-service/external-interfaces)
+            first)
         200)
```

Comment:
```suggestion
(http/no-cache-transit-response
        (as-> (assoc ts ::t-service/operation-area	        
                      (places/fetch-transport-service-operation-area db id)) ts	                      	           
            (first (netex-util/append-ote-netex-urls (vector ts) config db ::t-service/external-interfaces)))
            200)
```

Reply:
Thanks! 
However could you elaborate the reasoning for this suggestion? 
I guess this boils down to personal preference, initial motivation for current implementation was to have less nesting which usually simplifies reading

Reply:
Less nesting is always better but the single row "first" call is ugly for my eyes :)